packages/scandipwa/src/route/LoginAccount/LoginAccount.container.js

```diff
@@ -61,16 +66,28 @@ export class LoginAccountContainer extends MyAccountOverlayContainer {
         const {
             setHeaderState,
             toggleBreadcrumbs,
+            showErrorNotification,
             location: {
                 state: {
-                    isFromEmailChange = false
+                    isFromEmailChange = false,
+                    isFromLocked = false
                 } = {}
             }
         } = this.props;
 
-        if (isSignedIn() && !isFromEmailChange) {
+        const isLocked = BrowserDatabase.getItem(ISLOCKED);
+
+        if (isLocked || isFromLocked) {
+            console.log('called');
+            const message = 'The account sign-in was incorrect or your account is disabled temporarily.'
+            + 'Please wait and try again later.';
+
+            showErrorNotification(message);
+            BrowserDatabase.deleteItem(ISLOCKED);
```

Comment:
Can't you show error notification after user gets logged out due to lock instead of storing value in islocked then outputting error message on Login page and then removing islocked... seems kind of unneeded to relay on islocked variable.